provider/cluster_rosa_classic_resource.go

```diff
@@ -808,13 +813,90 @@ func (r *ClusterRosaClassicResource) validateAccountRoles(ctx context.Context, s
 
 	return nil
 }
+
+// validateOperatorRolePolicies ensures that the operator role policies are
+// compatible with the requested cluster version
+func (r *ClusterRosaClassicResource) validateOperatorRolePolicies(ctx context.Context, state *ClusterRosaClassicState, version string) error {
```

Comment:
Due to that conversation - https://redhat-internal.slack.com/archives/C03RNFXM72M/p1686160967085649 I don't think we need to add that validation code

Reply:
This should also be removed as a part of https://issues.redhat.com/browse/OCM-2323